includes/class-wc-payments.php

```diff
@@ -906,7 +908,13 @@ public static function ajax_init_platform_checkout() {
 
 		$platform_checkout_host = defined( 'PLATFORM_CHECKOUT_HOST' ) ? PLATFORM_CHECKOUT_HOST : 'http://host.docker.internal:8090';
 		$url                    = $platform_checkout_host . '/wp-json/platform-checkout/v1/init';
-		$body                   = [
+
+		if ( class_exists( 'RoutesController' ) ) {
```

Comment:
@waclawjacek I have tested with some versions and added these checks as per your feedback.
The current minimum version of `WooCommerce` is `5.0.0` which has the `Package` class but not `RoutesController`. So I added these checks and used 'wc/store' as the fallback here :eyes:

Reply:
Actually found an issue with this check. Fixing that now. :eyes:

Reply:
Even if the `RoutesController` class exists, in older versions it may not be registered in the container which throws an error. Added a try-catch [here](https://github.com/Automattic/woocommerce-payments/pull/4088/commits/ef4a7cda9c8907bb5f4966fb6a94556115aa13e1) to safely handle this scenario.